### PR TITLE
Ui

### PR DESCRIPTION
**Pull request by yashjk**

### CODE REVIEW

File: feature/kanban_board/client/components/App.jsx [hunk 1]
```diff
@@ -1,23 +1,91 @@
...
+      columnTitle: ''
+    }
   }
 
+  componentDidMount(){
+    this.props.fetchColumnList(this.createColumnList)
+  }
+  
+  createColumnList = () => {
+    console.log("inside callback")
```
Comment (srijan-kapoor) [inline feature/kanban_board/client/components/App.jsx]: Please do not leave `console` statements in code.

File: feature/kanban_board/client/components/App.jsx [hunk 2]
```diff
@@ -1,23 +1,91 @@
...
+  handleCancel = () => {
+    this.setState({columnTitle: ''})
+  }
+  handleChange = e => {
+    this.setState({[e.target.name] : e.target.value})
+  }
+  
   render() {
-    
+    console.log(this.props)
```
Comment (srijan-kapoor) [inline feature/kanban_board/client/components/App.jsx]: Same here

File: feature/kanban_board/client/components/Column.jsx [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            </div>
+        ) 
+    }
+}
+
+const mapStateToProps = store => store
+
+export default compose(DropTarget("note" , {} , collect), 
+    connect(mapStateToProps , {createNote , fetchNoteList , updateNote}))
+    (Column);
```
Comment (srijan-kapoor) [inline feature/kanban_board/client/components/Column.jsx]: Try to keep an empty line at the end of each file.

**Review by srijan-kapoor — CHANGES_REQUESTED** [review]
Looks good. Made some minor suggestions.

1. Make sure that changes from previous PRs do not get reflected in the subsequent pull-requests.
2. Use the `assignee` feature instead of `reviewers` when assigning PR for review.

File: feature/kanban_board/client/components/App.jsx (hunk 1 above)
Comment (yashjk) [reply feature/kanban_board/client/components/App.jsx]: resolved

File: feature/kanban_board/client/components/App.jsx (hunk 2 above)
Comment (yashjk) [reply feature/kanban_board/client/components/App.jsx]: resolved

File: feature/kanban_board/client/components/Column.jsx (hunk 3 above)
Comment (yashjk) [reply feature/kanban_board/client/components/Column.jsx]: resolved

**Review by srijan-kapoor — CHANGES_REQUESTED** [review]
Please rebase and fix conflicts with the base branch.

**Pull request merged**